src/main/fc/fc_init.c

```diff
@@ -533,6 +534,10 @@ void init(void)
     servosFilterInit();
 #endif
 
+#ifdef USE_PINIO
+    pinioInit(pinioConfig());
```

Comment:
Maybe pinioInit() can access PG internally ...

Reply:
No, it's better to separate it like this.